tests/test_circuit_ids.py

```diff
@@ -19,7 +19,7 @@ def _multi_index():
 
 
 class TestCircuitNodeId:
-    def setup(self):
+    def setup_method(self):
```

Comment:
why is the name changed?

Reply:
`setup` was specific to nosetests and pytest 7.2.0 raises now a warning if it is used:

```
PytestRemovedIn8Warning: Support for nose tests is deprecated and will be removed in a future release.
  tests/test_morph.py::TestMorphHelper::test_get_filepath is using nose-specific method: `setup(self)`
  To remove this warning, rename it to `setup_method(self)`
  See docs: https://docs.pytest.org/en/stable/deprecations.html#support-for-tests-written-for-nose
    fixture_result = next(generator)
```